### DISCUSSION TITLE
我的为何又这样了？

### DISCUSSION THREAD

**Discussion by xintie2023** [share121/inter-knot#1824] [其他]
我怎么进不去啊怎么这这么回事啊！
![Screenshot_20240813_111934_mark via](https://github.com/user-attachments/assets/09fb709e-7eb3-4e44-b065-6f75a225901c)

**Comment by justafish8787** [comment]
换成用x浏览器试试

**Reply by xintie2023** [reply]
谢谢成功了。

**Comment by cooperatior** [comment]
使用魔法

**Comment by cooperatior** [comment]
加速器

**Comment by Kamisato100** [comment]
我也这样，更新绳网助手也解决不了，最后是用 https://www.xn--5o0anh.top/ 这个域名进来才解决的

**Comment by Kamisato100** [comment]
就是[绳网.top](https://www.xn--5o0anh.top/)

**Comment by jzx-176** [comment]
Bili bili上别的UP主出了教程，下个watt加速
